manifests/kots-config.yaml

```diff
@@ -802,11 +802,14 @@ spec:
             [More info](https://docs.carto.com/carto-self-hosted/guides/use-workload-identity-in-gcp).
           type: bool
           default: "0"
-        - name: googleWorkloadIdentityEmail
-          title: Google workload identity email
+        - name: k8sWorkloadIdentityServiceAccountName
+          title: Kubernetes workload identity service account name
           when: '{{repl (ConfigOptionEquals "enableGoogleWorkloadIdentity" "1") }}'
           required: true
           type: text
+          help_text: |-
+            The kubernetes service account name you have created in the carto namespace.
+            Remember to annotate this SA with the GCP service account email.
```

Comment:
We'll have to add a link to the documentation and update the existing doc :)

Reply:
yep, sure!